proto/ptypes/replica.proto

```diff
@@ -5,70 +5,70 @@ package ptypes;
 import "google/protobuf/empty.proto";
 
 service ReplicaService {
-    rpc ReplicaCreate(ReplicaCreateRequest) returns (Replica) {}
-    rpc ReplicaDelete(google.protobuf.Empty) returns (google.protobuf.Empty) {}
-    rpc ReplicaGet(google.protobuf.Empty) returns (Replica) {}
-    rpc ReplicaOpen(google.protobuf.Empty) returns (Replica) {}
-    rpc ReplicaClose(google.protobuf.Empty) returns (Replica) {}
-    rpc ReplicaReload(google.protobuf.Empty) returns (Replica) {}
-    rpc ReplicaRevert(ReplicaRevertRequest) returns (Replica) {}
-    rpc ReplicaSnapshot(ReplicaSnapshotRequest) returns (Replica) {}
-    rpc ReplicaExpand(ReplicaExpandRequest) returns (Replica) {}
-    rpc DiskRemove(DiskRemoveRequest) returns (Replica) {}
-    rpc DiskReplace(DiskReplaceRequest) returns (Replica) {}
-    rpc DiskPrepareRemove(DiskPrepareRemoveRequest) returns (DiskPrepareRemoveReply) {}
-    rpc DiskMarkAsRemoved(DiskMarkAsRemovedRequest) returns (Replica) {}
-    rpc RebuildingSet(RebuildingSetRequest) returns (Replica) {}
-    rpc RevisionCounterSet(RevisionCounterSetRequest) returns (Replica) {}
+    rpc CreateReplica(CreateReplicaRequest) returns (Replica) {}
```

Comment:
Why change the name?

Reply:
Since the old name doesn't follow the naming convention:
https://cloud.google.com/apis/design/naming_convention#method_names

Reply:
Since the old method name didn't follow the golang naming convention. I was thinking we can correct both together.